marimo/_plugins/ui/_impl/table.py

```diff
@@ -273,3 +284,10 @@ def get_column_summaries(self, args: EmptyArgs) -> ColumnSummaries:
             )
 
         return ColumnSummaries(summaries)
+
+    def sort_values(self, args: SortValuesArgs) -> JSONType:
+        return (
+            self._unfiltered_manager.sort_values(args.by, args.descending)
```

Comment:
if `by` is allowed to be optional? can we pull the optional check to this function - so its required downstream (which should clean up the code)

Reply:
good idea